helper/chunkify_test.js

```diff
@@ -0,0 +1,40 @@
+
+function chunkify(message, maxChars){
+    let lines = message.split(/\n/);
+    let chunks = [];
+    let numChars = 0;
+    for (let i = 0; i < lines.length; i++){
+        numChars = lines[i].length;
+        if (numChars < maxChars){
+            chunks.push(lines[i]);

```

Comment:
I think this wouldn't combine multiple lines in to a single chunk? But leave it for now and see the behavior in testing

Reply:
Attempted to resolve in latest fix